### PR TITLE
Feature/settings page

### PR DESCRIPTION
**Pull request by CDRussell**
Asana Issue URLs: 
- Settings ticket: https://app.asana.com/0/414730916066338/492750258120871
- About Us ticket: https://app.asana.com/0/414730916066338/364465928695401

## Description
Adds a settings page, accessible from the overflow menu on BrowserActivity. 

### Settings Activity
The settings page shows the current app version, and provides a means to navigate to the about us activity.

### About Us Activity
Provides some static text about DDG. Also includes a link to view more information about this, and uses the `BrowserActivity` to navigate to the url if clicked.

## Steps to Test this PR:
1. Use the overflow menu to go to settings
1. Ensure version is shown and correct
1. Click to learn more about DDG
1. Ensure the about us page renders correctly (and is scrollable if the content goes off screen)
1. Click the URL to view more; ensure the app handles the page load (and not any default system browser)

### CODE REVIEW

File: app/src/main/res/layout/content_about_duck_duck_go.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/ddgLogo"
+                tools:text="Long description would go here\n\nLots and lots of words" />
+
+        </LinearLayout>
+
+    </ScrollView>
+
+    <TextView
+        android:id="@+id/learnMoreLink"
```
Comment (subsymbolic) [inline app/src/main/res/layout/content_about_duck_duck_go.xml]: The link is meant to be part of the text. Can we remove the border and put it inside the main LinearLayout of the scrollview? It's ok if the user needs to scroll to see it.

**Comment by subsymbolic** [comment]
I create a somewhat related ticket for allowing access to Settings on the Home screen https://app.asana.com/0/414730916066338/513409707430058. Feel free to do it here or separately - whatever is easiest for you.

**Comment by brindy** [comment]
This doesn't look right, but could be my low density emulator (480x800 mdpi running Lollipop) - will try on some other emulators too

screenshot_1515159033

**Comment by brindy** [comment]
I tried this on a low resolution running a newer version of Android it rendered fine.

**Pull request merged**